docs/src/pages/component-demos/tabs/ScrollableTabsButtonForce.js

```diff
@@ -0,0 +1,102 @@
+// @flow weak
+/* eslint-disable react/no-multi-comp */
+
+import React, { Component, PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import customPropTypes from 'material-ui/utils/customPropTypes';
+import Paper from 'material-ui/Paper';
+import { Tabs, Tab } from 'material-ui/Tabs';
+import PhoneIcon from 'material-ui-icons/Phone';
+import FavoriteIcon from 'material-ui-icons/Favorite';
+import PersonPinIcon from 'material-ui-icons/PersonPin';
+import HelpIcon from 'material-ui-icons/Help';
+import ShoppingBasket from 'material-ui-icons/ShoppingBasket';
+import ThumbDown from 'material-ui-icons/ThumbDown';
+import ThumbUp from 'material-ui-icons/ThumbUp';
+
+const TabContainer = (props) => (
+  <div style={{ padding: 20 }}>
+    {props.children}
+  </div>
+);
+
+TabContainer.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+const styleSheet = createStyleSheet('ScrollableTabsButtonForce', (theme) => ({
+  root: {
+    width: 500,
+  },
+  appBar: {
+    backgroundColor: theme.palette.background.appBar,
+  },
+}));
+
+export default class ScrollableTabsButtonForce extends Component {
+  static contextTypes = {
+    styleManager: customPropTypes.muiRequired,
+  };
+
+  state = {
+    index: 0,
+  };
+
+  handleChange = (event, index) => {
+    this.setState({ index });
+  }
+
+  render() {
+    const classes = this.context.styleManager.render(styleSheet);
+
+    return (
+      <Paper className={classes.root}>
+        <div className={classes.appBar}>
+          <Tabs
+            index={this.state.index}
+            onChange={this.handleChange}
+            scrollable
+            scrollButtons={'on'}
+            textColor="accent"
+          >
+            <Tab
+              label="Item One"
```

Comment:
Isn't what we are looking for, something rigid and opinionated to focus on something else?